src/frontend/Sider/Personoversikt/Behandlingsoversikt/OpprettNyBehandling/OpprettOrdinærBehandling.tsx

```diff
@@ -80,11 +99,31 @@ const OpprettOrdinærBehandling: React.FC<Props> = ({
             settLaster(false);
             return;
         }
+
+        let nyeOpplysninger: NyeOpplysningerMetadata | undefined = undefined;
+        if (årsak === BehandlingÅrsak.NYE_OPPLYSNINGER) {
+            if (!kilde) {
+                settFeilmelding(lagFeilmelding('Kilde for nye opplysninger må settes'));
+                settLaster(false);
+                return;
+            }
+            if (!endringer) {
+                settFeilmelding(lagFeilmelding('Endringer ved nye opplysninger må settes'));
+                settLaster(false);
+                return;
+            }
+            nyeOpplysninger = {
+                kilde: kilde,
+                endringer: endringer,
+                beskrivelse: beskrivelse,
+            };
+        }
```

Comment:
Gjerne trekk validering ut i en egen funksjon. 

Det vi har gjort en del andre steder er å returnere feilmelding som et object med keys for de ulike feltene ala: 

```
feilmelding = {
   kilde: 'Feilmelding for kilde',
   ...
}
```

Da kan man bruke de innebygde feilmeldingene som finnes for de ulike skjemakomponentene i aksel med `error={feilmelding.kilde}`